interfaces/builtin/daemon_notify.go

```diff
@@ -64,7 +64,7 @@ func (iface *daemoNotifyInterface) AppArmorConnectedPlug(spec *apparmor.Specific
 		// must be an absolute path or an abstract socket path
 		return fmt.Errorf("cannot use %q as notify socket path: not absolute", notifySocket)
 	}
-	illegalChars := `?*[]{}^"`
+	illegalChars := apparmorAARE
```

Comment:
Thank you for doing this, how about moving those to `apparmor.AARE` (as a const), I believe we have those in one other place as well.

Reply:
I moved this into apparmor as a helper now.